handbook/customer-success/README.md

```diff
@@ -0,0 +1,228 @@
+
+## Responsibilities
+
+#### First-line response
+
+Customer Success is responsible for the first response to messages in private customer Slack channels.
+
+Customer Support Engineers are responsible for the first response to Slack messages in the [#fleet channel](https://osquery.slack.com/archives/C01DXJL16D8) of osquery Slack, and other public Slacks. 
+
+The 24/7 on-call is responsible for alarms related to fleetdm.com and Fleet Managed Cloud, as well as delivering 24/7 support for Fleet Ultimate customers.
+
+We respond within 1-hour during business hours and 4 hours outside business hours. Note that we do not need to have answers within 1 hour -- we need to at least acknowledge and collect any additional necessary information, while researching/escalating to find answers internally.
+
+> Response SLAs help us measure and guarantee the responsiveness that a customer [can expect](https://fleetdm.com/handbook/company#values) from Fleet.  But SLAs aside, when a Fleet customer has an emergency or other time-sensitive situation ongoing, it is Fleet's priority to help them find them a solution quickly.
+
+### Onboard a customer support engineer
+What do you do every day? What does the path to success look like in this role and what can you do to contribute quickly at Fleet? To onboard a customer support engineer at Fleet it's important to understand the [continued training needed](https://docs.google.com/document/d/1GB8i_VMaFxeb9ipLock9MVWGJ2RqIW8lZ5n3MLiXG4s/edit).
+
+### Onboard a customer success manager
+What do you do every day? What does the path to success look like in this role and what can you do to contribute quickly at Fleet? To onboard a customer success manager at Fleet it's important to understand the [continued training needed](https://docs.google.com/document/d/1itrBeztwjK253Q548wbveVWdDaldBYCEOS6Cbz5Z4Uc/edit).
+
+### Onboard a customer solutions architect
+What do you do every day? What does the path to success look like in this role and what can you do to contribute quickly at Fleet? To onboard a customer solutions architect at Fleet it's important to understand the [continued training needed](https://docs.google.com/document/d/1G26Aqmn4tSKa7s0jMcSRqNTtz6h47Tvf8Ddi2-cP1ek/edit#heading=h.2i16pc77rnb7).
+
+### Onboard an infrastructure engineer
+What do you do every day? What does the path to success look like in this role and what can you do to contribute quickly at Fleet? To onboard a customer solutions architect at Fleet it's important to understand the [continued training needed](https://docs.google.com/document/d/1G26Aqmn4tSKa7s0jMcSRqNTtz6h47Tvf8Ddi2-cP1ek/edit#heading=h.2i16pc77rnb7).
+
+
+## Rituals
+
+<rituals :rituals="rituals['handbook/customers/sales.rituals.yml']"></rituals>
```

Comment:
@Sampfluger88 I'm wondering: "why is it named sales.rituals.yml"?  

Let's rename it to customer-success.rituals.yml.

@Sampfluger88 Will leave that to you, with the other requested changes I left, then feel free to merge.